README.md

```diff
@@ -1 +1,20 @@
 # health-tracker-backend
+
+## Lab 32
```

Comment:
For this Lab you'll need to have a Controller with a GET and POST route to save the EXERCISE to the DB.